src/Calculator/GraphingSettings.xaml.cpp

```diff
@@ -0,0 +1,220 @@
+                    m_height = m_width / m_currentAspectRatio;
+                    double bounds = m_height / 2;
+                    m_yMin = -bounds;
+                    m_yMax = bounds;
+                    m_ParentGrapher->YAxisMin = m_yMin;
+                    m_ParentGrapher->YAxisMax = m_yMax;
+                }
+
+                if (heightChanged)
+                {
+                    m_height = m_yMax - m_yMin;
+                    m_width = m_height * m_currentAspectRatio;
+                    double bounds = m_width / 2;
+                    m_xMin = -bounds;
+                    m_xMax = bounds;
+                    m_ParentGrapher->XAxisMin = m_xMin;
+                    m_ParentGrapher->XAxisMax = m_xMax;
+                }
+
+                if (widthChanged || heightChanged)
+                {
+                    x_min->Text = m_xMin.ToString();
+                    x_max->Text = m_xMax.ToString();
+
+                    y_min->Text = m_yMin.ToString();
+                    y_max->Text = m_yMax.ToString();
+
+                    m_ParentGrapher->SetDisplayRanges(m_xMin, m_xMax, m_yMin, m_yMax);
+                }
+            }
+        }
+    }
+    catch (const std::exception&)
+    {
+    }
+}
+
+void CalculatorApp::Controls::GraphingSettings::OnLoaded(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
+{
+    m_ParentGrapher->GetDisplayRanges(&m_xMin, &m_xMax, &m_yMin, &m_yMax);
+
+    // Now we can load our UX
+    x_min->Text = m_xMin.ToString();
+    x_max->Text = m_xMax.ToString();
+
+    y_min->Text = m_yMin.ToString();
+    y_max->Text = m_yMax.ToString();
+
+    switch ((Graphing::EvalTrigUnitMode)m_ParentGrapher->TrigUnitMode)
+    {
+    case Graphing::EvalTrigUnitMode::Degrees:
+    {
+        Degrees->IsChecked = true;
+    }
+    break;
+    case Graphing::EvalTrigUnitMode::Radians:
+    {
+        Radians->IsChecked = true;
+    }
+    break;
+    case Graphing::EvalTrigUnitMode::Grads:
+    {
+        Gradians->IsChecked = true;
+    }
+    break;
+
+    case Graphing::EvalTrigUnitMode::Invalid:
+    {
+        OutputDebugString(L"GraphingSettings::GraphingCalculator_DataContextChanged Unsupported TrigUnitMode\r\n");
+        m_ParentGrapher->TrigUnitMode = (int)Graphing::EvalTrigUnitMode::Radians;
+    }
+    }
+
+    m_width = m_xMax - m_xMin;
+    m_height = m_yMax - m_yMin;
+    m_currentAspectRatio = m_height / m_width;
+}
+
+void GraphingSettings::OnLosingFocus(Windows::UI::Xaml::UIElement ^ sender, Windows::UI::Xaml::Input::LosingFocusEventArgs ^ args)
+{
+    UpdateExtents(sender);
+}
+
+void GraphingSettings::OnKeyDown(Platform::Object ^ sender, Windows::UI::Xaml::Input::KeyRoutedEventArgs ^ e)
+{
+    if (e->Key == Windows::System::VirtualKey::Enter)
+    {
+        UpdateExtents(sender);
+    }
+}
+
+void CalculatorApp::Controls::GraphingSettings::OnPreserveAspectRatioClicked(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
```

Comment:
Nit comment for here and throughout the PR. Style of the project is to not use the full namespaces in .cpp files
```suggestion
void GraphingSettings::OnPreserveAspectRatioClicked(Object ^ sender, RoutedEventArgs ^ e)
```

Reply:
FIxed, VS autogenerated method I forgot to cleanup.